test/test_list.py

```diff
@@ -0,0 +1,41 @@
+from numpy.testing import assert_almost_equal
+
+from pyproj import (
+    get_angular_units_map,
+    get_ellps_map,
+    get_prime_meridians_map,
+    get_proj_operations_map,
+    get_units_map,
+)
+from pyproj._list import Unit
+
+
+def test_units_map():
+    units_map = get_units_map()
+    assert isinstance(units_map["m"], Unit)
+    assert units_map["m"].id == "m"
+    assert units_map["m"].name == "Meter"
+
+
+def test_angular_units_map():
+    ang_map = get_angular_units_map()
+    assert isinstance(ang_map["deg"], Unit)
+    assert ang_map["deg"].id == "deg"
+    assert ang_map["deg"].name == "Degree"
+
+
+def test_get_ellps_map():
+    ellps_map = get_ellps_map()
+    assert ellps_map["WGS84"]["description"] == "WGS 84"
+    assert_almost_equal(ellps_map["WGS84"]["a"], 6378137.0, decimal=1)
+    assert_almost_equal(ellps_map["WGS84"]["rf"], 298.257223563, decimal=1)
```

Comment:
The "rf" factor should be tested to a few more decimal places.  Several of them have "rf" factors around 298.25.

Reply:
This is mostly just checking that it is in the correct format. It does not attempt to check values as that should be handled by PROJ.